aspnet/signalr/overview/performance/scaleout-with-redis.md

```diff
@@ -74,11 +74,11 @@ Download the Ubuntu ISO from [http://www.ubuntu.com](http://www.ubuntu.com/).
 
 In Hyper-V, add a new VM. In the **Connect Virtual Hard Disk** step, select **Create a virtual hard disk**.
 
-![Window titled New Virtual Machine Wizard. 'Connect Virtual Hard Disk' is highlighted and 'Create a virtual hard disk' is selected.](scaleout-with-redis/_static/image2.png)
+![Screenshot of a window titled New Virtual Machine Wizard and Connect Virtual Hard Disk and Create a virtual hard disk are selected.](scaleout-with-redis/_static/image2.png)
 
 In the **Installation Options** step, select **Image file (.iso)**, click **Browse**, and browse to the Ubuntu installation ISO.
 
-![Window titled New Virtual Machine Wizard. 'Installation Options' is highlighted. 'Install...from a boot CD/DVD-ROM' and 'Image file (.iso):' are selected"](scaleout-with-redis/_static/image3.png)
+![Screenshot that shows a window titled New Virtual Machine Wizard Installation Options is highlighted Install from a boot C D D V D ROM and Image file dot i s o are selected](scaleout-with-redis/_static/image3.png)
```

Comment:
How about, "Screenshot that shows New Virtual Machine Wizard on the Installation Options page. Install an O S from a boot C D D V D ROM and Image file dot iso are selected."